### ISSUE TITLE
Post listing jumping around on mouseover (Safari)

### ISSUE THREAD

**Issue by Joshfindit** [sharu725/karna#2]
Specs:

```
OSX 10.12.1
Safari 10.0.1 (12602.2.14.0.7)
Extensions: 1Password and Evernote Web Clipper
```

Note: Tested in Chrome, and it doesn't happen there.

![karna-issue](https://cloud.githubusercontent.com/assets/1937155/21018924/242810d6-bd45-11e6-8b03-2c3826930c0c.gif)

**Comment by sharu725** [comment]
I'm guessing this is because of the box-shadow that changes when hovered over cards. You can try removing the `box-shadow` on hover and try again to see if the same thing happens.

**Comment by Joshfindit** [comment]
That does stop the jumping, but the left column is still mis-aligned.. 
Maybe it's a combination of the two?
![screenshot 2016-12-08 15 35 12](https://cloud.githubusercontent.com/assets/1937155/21024756/060f6272-bd5c-11e6-9488-11050c63a447.png)

**Comment by sharu725** [comment]
I still see a shadow in the first card. Have you tried removing that as well? Because there is box-shadow before hovering as well.

**Comment by Joshfindit** [comment]
Do you see this as something that users should resolve, or a bug with the theme?

**Comment by sharu725** [comment]
I use Chrome and  Firefox which are widely used internet browsers across the world. I try to make it compatible with those browsers. And I'm not willing to download another incapable browser (this includes old IEs ).

**Comment by Joshfindit** [comment]
That's your choice of course.
Safari is the fastest browser on OSX (right now), and also the only browser engine on iOS (and I've confirmed that the misalignment also happens on iOS)

If it helps, and if you're interested, I can work with you to debug this.

**Comment by sharu725** [comment]
@Joshfindit thank you. That would be great. I did some research meanwhile. This may be because I did not use webkit compatible CSS for box-shadows. I will change that and let's see if that solves the issue.

**Comment by sharu725** [comment]
Here it is [#d184007](https://github.com/sharu725/karna/commit/d1840070a2f803a7b494b8c38983c3f574d29f33)

**Comment by Joshfindit** [comment]
@sharu725 Thank you. I did apply those changes, but there was no effect on the issue.
I did some poking around, and pushed the hover shadow height to 60 pixels. It seems to me like the shadows of the top row are pushing against the edge of the container (except the shadows of the left column for some reason), but I'm certainly not good on CSS so I've included the recording for your analysis:

![karna issue shadow height to 60px](https://cloud.githubusercontent.com/assets/1937155/21206159/9a9b859a-c236-11e6-9bcc-5c448b9c4a59.gif)
